editor/components/block-list/block.js

```diff
@@ -421,6 +422,7 @@ export class BlockListBlock extends Component {
 								id={ block.uid }
 								isSelectionEnabled={ this.props.isSelectionEnabled }
 								toggleSelection={ this.props.toggleSelection }
+								createErrorNotice={ this.props.createErrorNotice }
```

Comment:
Code-wise it also seems cleaner to display the error within the block? This error does not apply to the whole page.

Reply:
Thank you for the feedback @iseulde, I will propose a version with a block-specific error 👍